techniques/about.html

```diff
@@ -49,73 +49,73 @@
+              <h2>WCAG について</h2>
+              <p>Web Content Accessibility Guidelines (WCAG) は、ウェブサイト、アプリケーション、及びその他のデジタルコンテンツを、障害のある人にとってアクセシブルにするための要件を提供している。WCAG の概要、関連する技術文書、教材については、<a href="https://www.w3.org/WAI/standards-guidelines/wcag/">WCAG 2 Overview</a> を参照。</p>
 
-          <p>There are additional resources that help you understand and implement WCAG. These Techniques are one type of resource. Others are explained in the <a href="https://www.w3.org/WAI/standards-guidelines/wcag/docs/">WCAG 2 Documents</a>.</p>
+          <p>WCAG を理解し実践するのに役立つ追加のリソースもある。このテクニック集は、そのリソースの一種である。その他のリソースについては <a href="https://www.w3.org/WAI/standards-guidelines/wcag/docs/">WCAG 2 Documents</a> で説明されている。</p>
           </section>
           <section id="introduction">
-            <h2>About Techniques</h2>
-              <p>The techniques provide guidance for web content  authors and evaluators on meeting WCAG success criteria. Techniques are examples of ways to meet WCAG.</p>
+            <h2>テクニック集について</h2>
+              <p>テクニック集は、ウェブコンテンツ制作者及び評価者が WCAG 達成基準を満たすにあたってのガイダンスを提供するものである。テクニック集は、WCAG を満たす方法の例である。</p>
               <section id="updated">
-              <h3>Updated periodically</h3>
-              <p>WCAG 2 itself is a stable document that does not change. The techniques for WCAG are updated periodically to cover more current best practices and changes in technologies and tools.</p>
+              <h3>定期的に更新される</h3>
+              <p>WCAG 2 自身は、変更されることのない安定した文書である。この WCAG テクニック集は、現在のベストプラクティス並びに技術及びツールの変化をより多く扱うために、定期的に更新される文書である。</p>
               </section>
               <section id="not-required">
-              <h3>Techniques are not required</h3>
-              <p>Techniques are informative—that means they are not required. The basis for determining conformance to WCAG 2 is the success criteria from the WCAG 2 standard—not the techniques.</p>
+              <h3>テクニックは必須要件ではない</h3>
+              <p>テクニックは参考情報であり、つまり必須要件ではない。WCAG 2 への適合を判断する基準は、WCAG 2 標準の達成基準であり、テクニックではない。</p>
               <div class="note">
-				<p class="note-title marker">Note</p>
+				<p class="note-title marker">注記</p>
 				<div>
-                <p>W3C cautions against requiring W3C's sufficient techniques. The only thing that should be required is meeting the WCAG 2 success criteria. To learn more, see: <a href="https://www.w3.org/WAI/standards-guidelines/wcag/faq/#techsnot">What would be the negative consequences of allowing <em>only</em> W3C's published techniques to be used for conformance to WCAG 2?</a> in the WCAG 2 FAQ</p>
+                <p>W3C は、W3C の十分なテクニックを要求しないよう警告する。要求されるべき唯一のものは、WCAG 2 の達成基準を満たすことである。より詳しくは、WCAG 2 FAQ における <a href="https://www.w3.org/WAI/standards-guidelines/wcag/faq/#techsnot">What would be the negative consequences of allowing <em>only</em> W3C's published techniques to be used for conformance to WCAG 2?</a> を参照。</p>
```

Comment:
ここも上記と同様で、https://waic.jp/translations/WCAG22/Understanding/understanding-techniques.html に揃えるなら: 

```suggestion
                <p>W3C は、W3C の十分なテクニックを必須としないように警告する。求められる唯一のことは WCAG 2 達成基準を満たすことである。より詳しくは、WCAG 2 FAQ における <a href="https://www.w3.org/WAI/standards-guidelines/wcag/faq/#techsnot">What would be the negative consequences of allowing <em>only</em> W3C's published techniques to be used for conformance to WCAG 2?</a> を参照のこと。</p>
```